.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Proposed Changes
+
+- Change
+- Update
+- Fix
+
+## Relates To
+
+- Issue #0
+
+## Notify
+
+- @octocat
+- @beetboxvm/octoteam
```

Comment:
@alexdesignworks we currently get notifications on the `drupalmel` slack channel so probably don't need this.

Anyone is welcome to join @ http://melbourne.drupal.org.au/